Makefile

```diff
@@ -1,54 +1,92 @@
-include cookbook/Makefile
 .SILENT:
 
 # Update PATH variable to leverage _bin directory
-export PATH := _bin:$(PATH)
+export PATH := .sandbox/bin:$(PATH)
 
 # Dependencies
-export K3D_VERSION := v4.2.0
+export DOCKER_VERSION := 20.10.3
+export K3S_VERSION := v1.20.2%2Bk3s1
 export KUBECTL_VERSION := v1.20.2
+export FLYTE_SANDBOX_IMAGE := flyte-sandbox:latest
 
-# Flyte cluster configuration variables
-FLYTE_CLUSTER_NAME := flyte
-FLYTE_CLUSTER_CONTEXT := k3d-$(FLYTE_CLUSTER_NAME)
-FLYTE_PROXY_PORT := 8001
+# Flyte sandbox configuration variables
+KUBERNETES_API_PORT := 51234
+FLYTE_PROXY_PORT := 51235
+FLYTE_SANDBOX_NAME := flyte-sandbox
 
 # Use an ephemeral kubeconfig, so as not to litter the default one
-export KUBECONFIG=$(HOME)/.k3d/kubeconfig-$(FLYTE_CLUSTER_NAME).yaml
+export KUBECONFIG=$(PWD)/.sandbox/data/config/kubeconfig
 
-# Helper to determine if a cluster is up and running - uses the ephemeral kubeconfig
-# as a flag
-IS_UP = $(shell [ -f $(KUBECONFIG) ] && echo true)
+# Module of cookbook examples to register
+EXAMPLES_MODULE := core
```

Comment:
Should this be to allow overrides?
```suggestion
export EXAMPLES_MODULE ?= core
```

Reply:
Should be able to “make register EXAMPLES_MODULE=plugins” with a make variable override. Should we still make it an environment variable?

Reply:
oh I didn't know that's a thing! Awesome

Reply:
```
> make register EXAMPLES_MODULE=plugins
Registering example workflows
docker run -it --rm --network flyte-sandbox -e MAKEFLAGS -e DOCKER_BUILDKIT=1 -e SANDBOX=1 -e FLYTE_HOST=flyte-sandbox:30081 -e FLYTE_AWS_ENDPOINT=http://flyte-sandbox:30084 --volumes-from flyte-sandbox -v /Users/jeev/Workspace/repos/flytesnacks:/mnt/src -w /usr/src flyte-sandbox:latest with-src-snapshot.sh make -C cookbook/plugins register
```